### PR TITLE
[script][common-items] Added messaging for successfully tapping tessera

### PR DESCRIPTION
**Pull request by Smaptastic**
Previously common-items.lic did not have the messaging given when successfully tapping a tessera. Added this to resolve an issue where tessera.lic would exit unexpectedly when both hands were full.

### CODE REVIEW

**Comment by Smaptastic** [comment]
The message received with the edit. Tap was recognized and messaging given that hands were full. Previously it would give an error and tessera would exit.
`[tessera]>get my tessera 

Samantaka climbed up some steps crudely crafted from gathered stones.

> 
You need a free hand to pick that up.
> 
[tessera]>tap my tessera 

You suddenly forget what you were doing.
> 
Samantaka went through an iron gate.

> 
Hands full, will try again later
Unpausing ["sanowret-crystal", "play"], tessera has finished.

--- Lich: sanowret-crystal unpaused.

--- Lich: play unpaused.
`

File: common-items.lic [hunk 1]
```diff
@@ -269,7 +269,8 @@ module DRCI
     /^You tap\s(?!into).*/, # The `.*` is needed to capture entire phrase. Methods parse it to know if an item is worn, stowed, etc.
     /^You (thump|drum) your finger/, # You tapped an item with fancy verbiage, ohh la la!
     /^The orb is delicate/, # You tapped a favor orb
-    /^You .* on the shoulder/ # You tapped someone
+    /^You .* on the shoulder/, # You tapped someone
+    /^You suddenly forget what you were doing/
```
Comment (MahtraDR) [inline common-items.lic:273]:
```suggestion
    /^You suddenly forget what you were doing/ # "tap my tessera" messaging when hands are full
```

**Comment by Raykyn55** [comment]
Works for me:
```
>;e echo DRCI.exists?('tessera')
--- Lich: exec1 active.
[exec1]>tap my tessera 
You suddenly forget what you were doing.
>
[exec1: true]
--- Lich: exec1 has exited.
```

**Comment by Smaptastic** [comment]
> >;e echo DRCI.exists?('tessera')

I like your funny words, magic man.

**Pull request merged**